test/verify/check-apps

```diff
@@ -103,12 +102,21 @@ class TestApps(packagelib.PackageCase):
         b.wait_visible(".pf-v5-c-empty-state")
 
         self.createAppStreamPackage("app-1", "1.0", "1")
+        self.createAppStreamPackage("app-2", "1.0", "1", install=True)
```

Comment:
app-2 is not checked anywhere in the test, and it may also affect the expectations of the existing checks. So something is missing  here.

Reply:
added a check for app-2